app/src/main/java/vn/mbm/phimp/me/newGallery.java

```diff
@@ -3698,7 +3729,7 @@ public boolean onLongClick(View v) {
 		ImageView imageSelector;
 	}
 
-	public static void resumeLocalPhoto(){
+	public void resumeLocalPhoto(){
```

Comment:
I don't think so. Everything is working fine. 
PS: I changed that because there was a merge conflict